TemporaryParser/TPRBParenthesisErrorNode.class.st

```diff
@@ -0,0 +1,28 @@
+Class {
+	#name : #TPRBParenthesisErrorNode,
+	#superclass : #TPRBEnglobingErrorNode,
+	#category : #'TemporaryParser-Nodes'
+}
+
+{ #category : #'as yet unclassified' }
+TPRBParenthesisErrorNode class >> error: aToken withNodes: aCollection [
+	| message |
+	message := ('(' = aToken value asString) ifTrue: [ ''')'' expected' ] 

```

Comment:
This method requires a nice comment too. Why so many ifs? what cases does it cover?